### PR TITLE
Fix cursors to support other formats than RGBA 32-bit

### PR DESCRIPTION
**Pull request by ArchDemons**
+ added support BGR8 cursor image format
* fixed idle cursor hotspot

### CODE REVIEW

File: src/toniarts/openkeeper/gui/Cursor.java [hunk 1]
```diff
@@ -19,17 +19,18 @@
...
 import java.nio.ByteBuffer;
 import java.nio.IntBuffer;
-import java.util.regex.Matcher;
 import toniarts.openkeeper.tools.convert.AssetsConverter;
+import toniarts.openkeeper.tools.convert.ConversionUtils;
 
 public class Cursor extends JmeCursor {
 
-    public int delay = 30;
+    public final int delay = 30;
```
Comment (tonihele) [inline src/toniarts/openkeeper/gui/Cursor.java]: Oooh, my bad, this should be a constant (final static)? No need to change it though, just spotted here :)

**Comment by tonihele** [comment]
Just wondering, has the possession cursor now a transparent background?

File: src/toniarts/openkeeper/gui/Cursor.java [hunk 2]
```diff
@@ -101,4 +101,27 @@ public Cursor(AssetManager assetManager, String Filename, int hotspotx, int hots
...
         this.setImagesData(image);
     }
+
+    private int getARGB(Format format, ByteBuffer data) {
+	int result = 0;
+
+	switch (format) {
+            case BGR8:
+                result = 0xFF000000 | (data.get() & 0xFF) | ((data.get() & 0xFF) << 8) | ((data.get() & 0xFF) << 16);
+                if (result == 0xFF00FF00) { // green only
```
Comment (ArchDemons) [inline src/toniarts/openkeeper/gui/Cursor.java]: check if green only color and change it to transparent

File: src/toniarts/openkeeper/gui/Cursor.java (hunk 1 above)
Comment (ArchDemons) [inline src/toniarts/openkeeper/gui/Cursor.java]: all not changeable must be static ! ^_^

Comment (tonihele) [inline src/toniarts/openkeeper/gui/Cursor.java]: For clarity & performance!

**Comment by ArchDemons** [comment]
We also have cursors in folder **Textures/cursor**. What we must do with them ?

**Comment by tonihele** [comment]
Hmm, what kind of stuff is there (I'm on a business trip, back on Wednesday evening, no access to resources)?

But other that this PR is ok, and a good job sticker :) Better than my suggestion to fiddle around with BufferedImages. Ok to merge?

**Comment by ArchDemons** [comment]
**Textures/cursor** contain Pointer, idle, press cursors. I think them forgot to remove.

You can merge.

**Comment by tonihele** [comment]
Maybe some old stuff. I also heard that DK II doesn't really use a cursor. Just a quad etc. following the mouse. Maybe they tried to use a cursor with these :)

**Pull request merged**